### DISCUSSION TITLE
PORT environment variable is not working

### DISCUSSION THREAD

**Discussion by dikhin** [open-webui/open-webui#6595] [General]
![image](https://github.com/user-attachments/assets/4c3d5c9d-ce41-4727-b5ec-2a90aab99887)
![image](https://github.com/user-attachments/assets/b47547c9-9e25-43eb-b707-f8ff7d32e331)

I have set PORT as an environment variable, however it is still served at the default port 8080.

**Comment by tjbck** [comment]
If you're using pip install, `--port` is used instead of env var.

**Reply by lcrownover** [reply]
This should probably be in the docs, spent a bit of time figuring out why the conda version of openwebui wasn't listening on the right port 😅

**Reply by dikhin** [reply]
Thanks tjbck, it works.
I agree with Icrownover, I now notice an additional info section in the docs for mentioning this.👍🏻

**Comment by oatmealm** [comment]
@tjbck What about other environment variables? Using `uvx` open-webui seems to ignore `.env` all together, even if it was sourced manually `source .env`, though it is reading  `.webui_secret_key` from the directory from which the command is issued...
